### PR TITLE
Add Play Framework 1.3

### PR DESCRIPTION
**Pull request by fdrouet**

### CODE REVIEW

File: play13.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    rm_rf "python" # we don't need the bundled Python for windows
+    rm Dir["*.bat"]
+    libexec.install Dir["*"]
+    bin.mkpath
+    chmod 0755, libexec+"play"
+    ln_s libexec+"play", bin
+  end
+
+  test do
+    # no test yet
```
Comment (DomT4) [inline play13.rb]: Could you possibly add a test down here if one is possible? If there isn't anything that can be done, remove the empty ` test do ` block. Thanks!

Comment (fdrouet) [inline play13.rb]: If I don't put test block, the jenkins job which check the formula syntax raise an error.
That is why I had to add one

Comment (DomT4) [inline play13.rb]: We can ignore the test bot's failure, if that's the only thing that fails.

**Comment by DomT4** [comment]
@BrewTestBot test this please
